.github/workflows/build-binaries.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         uses: PyO3/maturin-action@v1
         with:
           target: aarch64
-          args: --release --locked --out dist --features self-update
+          args: --release --locked --out dist --features production
```

Comment:
I think this and the reference on line 82 also need `self-update`, unless I'm misreading.